README.md

```diff
@@ -10,150 +10,148 @@
 
 A GitHub plugin for Mattermost. Supports GitHub SaaS and Enterprise versions.
 
+## Table of Contents
+
+ - [1. Audience](#1-audience)
+ - [2. License](#2-license)
+ - [3. About the GitHub Plugin](#3-about-the-github-plugin)
+ - [4. Before You Start](#4-before-you-start)
+ - [5. Configuration](#5-configuration)
+ - [6. Using the Plugin](#6-using-the-plugin)
+ - [7. Onboarding Your Users](#7-onboarding-your-users)
+ - [8. Slash Commands](#8-slash-commands)
+ - [9. Frequently Asked Questions](#9-frequently-asked-questions)
+ - [10. Development](#10-development)
+
 ![GitHub plugin screenshot](https://user-images.githubusercontent.com/13119842/54380268-6adab180-4661-11e9-8470-a9c615c00041.png)
 
-## Features
+## Audience
+
+This guide is intended for Mattermost System Admins setting up the GitHub plugin and Mattermost users who want information about the plugin functionality. For more information about contributing to this plugin, visit the [Development section](https://github.com/mattermost/mattermost-plugin-github#development).
+
+## License
+
+This repository is licensed under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0).
```

Comment:
Actually, I was suggesting to link to https://github.com/mattermost/mattermost-plugin-github/blob/master/LICENSE. 0/5 of what the better link target is.